### ISSUE TITLE
Missing alt text from WoS Source in Detailed Record

### ISSUE THREAD

**Issue by mdavismcd** [usnistgov/primoNDE#30]
<img width="738" height="276" alt="Image" src="https://github.com/user-attachments/assets/a0d4ac0c-5a14-48aa-b6fe-ad0493891a16" />
